packages/web/src/components/gcds-radio-group/gcds-radio-group.css

```diff
@@ -22,41 +22,53 @@
 
 @layer default {
   :host .gcds-radio {
-    max-width: 90%;
+    position: relative;
+    max-width: var(--gcds-radio-max-width);
+    min-height: calc(
+      var(--gcds-radio-input-height-and-width) - var(--gcds-radio-padding)
+    );
     font: var(--gcds-radio-font);
     color: var(--gcds-radio-default-text);
     margin: var(--gcds-radio-margin) !important;
+    padding: var(--gcds-radio-padding) 0 0;
```

Comment:
Same question about the extra `0 0` from above.

Reply:
The same comment - used the same token in the min-height calculation on line 28 which wouldn't
t be possible if I included the `0 0`.